src/test/ci/leaderboard.test.ts

```diff
@@ -0,0 +1,87 @@
+import assert from "assert";
+import { describe } from "mocha";
+import LeaderboardCommand, { LeaderboardType, LeaderboardDuration } from "../../commands/game_commands/leaderboard";
+import { getSqlDateString } from "../../helpers/discord_utils";
+import dbContext from "../../database_context";
+import MessageContext from "../../structures/message_context";
+import KmqMember from "../../structures/kmq_member";
+
+describe("getLeaderboardEmbeds", () => {
+    describe("all-time leaderboard", () => {
+        afterEach(async () => {
+            await dbContext.kmq("player_stats").del();
+        });
+
+        it("should have one page of entries", async () => {
+            const entries = Math.floor((Math.random() * 50) / 10) * 10;
```

Comment:
Tests shouldn't have randomness. If using randomness to account for edge case, the edge case should just be a different test